Cargo.toml

```diff
@@ -48,6 +48,7 @@ linux-static-hidraw = []
 linux-shared-libusb = []
 linux-shared-hidraw = []
 linux-native = ["dep:udev", "dep:nix"]
+linux-native-basic-udev = ["dep:basic-udev", "dep:nix"]
```

Comment:
I think its not necessary to create a `backend` feature for just swapping out a compatible alternative implementation of a dependency. Consider just creating a udev and basic-udev feature and selecting the normal udev feature by default.

Reply:
How would that work? Wouldn't that require people who select `linux-native` to also select the new `udev` feature? Or that would force users to all opt-in to the new `udev` crate, even those who are compiling for e.g. illumos? My concern is that as long as you select the `udev` crate, it will run the pkgconfig script at build time, which may impact users who don't want to depend on udev.

Reply:
Ah right, my mistake. Then lets keep it like it is!

On another note maybe you can add the new linux-native-basic-udev feature as one of the CI  test targets?

Reply:
Sure. I added it to CI. I added both an Ubuntu target as well as one that cross-compiles to arm-musleabihf from Windows just to make sure that it does function as expected.

Reply:
...right. `cargo test` probably isn't a good idea to run when cross-compiling.